.env.example

```
@@ -0,0 +1,100 @@
+# -------------------------------------
+# MindsDB Server Connection Details
+# -------------------------------------
+# Use these for a remote or authenticated local MindsDB instance
+# MINDSDB_PROTOCOL=https
+# MINDSDB_HOST=your.mindsdb.cloud.instance
+# MINDSDB_PORT=443
+# MINDSDB_USER=your_email@example.com
+# MINDSDB_PASSWORD=your_mindsdb_password
+
+# Use these for a default local MindsDB instance (no auth)
+MINDSDB_PROTOCOL=http
+MINDSDB_HOST=127.0.0.1
+MINDSDB_PORT=47334
+MINDSDB_USER=
+MINDSDB_PASSWORD=
+
+# -------------------------------------
+# Handlers to be Tested
+# Comma-separated list of handler names.
+# -------------------------------------
+HANDLERS_TO_TEST=postgres,s3,github,databricks,bigquery,mariadb,mysql
+
+# =====================================
+# Data Source Credentials
+# =====================================
+
+# --- PostgreSQL ---
+PG_SOURCE_HOST=samples.mindsdb.com
```

We should make as many of these DB params as possible just work out of the box. As it stands users have to manually copy this file

---

Out of the box will mean ? 
I believe users will need to test 1 or two handlers otherwise we will have set of priotized handlers to execute in ci pipeline. 

could you provide example of out of the box you mean ?  are we already doing this somewhere ?

---

Out of the box = without the user having to set anything up.

So for DB types where we have a sample DB, we can default to the sample DB's user/pass/host etc. If there is a sane default value we shouldn't force the user to copy an env file etc.

---

Absolutely right, can you provide those defaults for the missing ones in the list s3,github? I actually like the idea of default target datasource
When this task was give the request was that this can be configured by users, providing credentials for a target datasource should not be seen as forcing the user. What if the user needs to test against different target datasource? 

**What I will do**:
keep the configuration options and have the test use default credentials if not config is provided.

---

the test now uses default credentials if no config is provided. I am resolving this comment